LBD2024G06TP1Script.sql

```diff
@@ -120,14 +144,45 @@ CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`LINEAS_PARTES` (
 ENGINE = InnoDB
 DEFAULT CHARACTER SET = utf8;
 
+-- Update with new Tables 
+
+CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`VEHICULOS` (
+  `idVEHICULO` INT(11) NOT NULL,
+  `tipo` VARCHAR(45) NOT NULL,
+  `modelo` VARCHAR(45) NOT NULL,
+  `funcion` VARCHAR(45) NULL DEFAULT NULL,
+  PRIMARY KEY (`idVEHICULO`),
+  UNIQUE INDEX `idVEHICULO_UNIQUE` (`idVEHICULO` ASC) VISIBLE)
+ENGINE = InnoDB
+DEFAULT CHARACTER SET = utf8;
+
+CREATE TABLE IF NOT EXISTS `LBD2024G06AGROSA`.`MAQUINAS_POR_PARTES` (
+  `idVEHICULO` INT(11) NOT NULL,
+  `idPARTE` INT(11) NOT NULL,
+  PRIMARY KEY (`idVEHICULO`, `idPARTE`),
+  INDEX `idPARTE` (`idPARTE` ASC) VISIBLE,
```

Comment:
We must use a compound index here... it doenst make sense to only search for one field. but the compund index is made by the PK! So we dont need to add other ones :)